spec/unit/lib/cadence_spec.rb

```diff
@@ -206,6 +205,35 @@ class TestStartWorkflow < Cadence::Workflow
       end
     end
 
+    describe '.get_workflow_history' do
+      let(:cadence_response) do
+        CadenceThrift::StartWorkflowExecutionResponse.new(runId: 'xxx')
+      end
+      let(:response_mock) { double }
+      let(:history_mock) { double}
+      let(:event_mock) { double('EventMock', :eventId=> 1, :timestamp => Time.now.to_f, :eventType => 'ActivityTaskStarted', :eventAttributes => '') }
+  
+      before do
+        allow(client).to receive(:start_workflow_execution).and_return(cadence_response)
+        allow(history_mock).to receive(:events).and_return([event_mock])
+        allow(response_mock).to receive(:history).and_return(history_mock)
+      end
+
+      it 'wraps client get_workflow_execution_history' do
+          run_id = described_class.start_workflow(TestStartWorkflow, 42, options: { workflow_id: '123' })
```

Comment:
Hmm… why do you need to call this if the client interaction is stubbed anyways? I don't think there's any need for this since your method is calling the client directly